packages/flutter/test/widgets/custom_painter_test.dart

```diff
@@ -357,13 +357,13 @@ void _defineTests() {
         new TestSemantics.rootChild(
           id: 1,
           previousNodeId: -1,
-          nextNodeId: expectedId,
+          nextNodeId: -1,
```

Comment:
This looks odd now. Does -1 mean "not specified"? Can we make this the default so you don't have to write it out in your tests?

Reply:
Done (see https://github.com/flutter/flutter/pull/15537/commits/b2e2aa0c6b2eb4bfe51ba1096e861ed919b39a34)